site/src/js/library/modules/callbacks/BlogDetailWidget.js

```diff
@@ -7,14 +8,7 @@
  * @returns {object} Updated widget data
  */
 const blogDetail_data_changeDate = function (data) {
-  // Update article published date from "X days ago" to actual date
-  if (data.newsarticle.publisheddate != '') {
-    const date = new Date(data.newsarticle.Publish_x0020_Date);
-    const options = { year: 'numeric', month: 'long', day: 'numeric' };
-    data.newsarticle.publisheddate = date.toLocaleDateString('en-US', options);
-  }
-
-  return data;
+  formatting.dateFormat(data);
```

Comment:
Instead of passing the whole data object to the `dateFormat` function, let's just pass the actual date field, `data.newsarticle.Publish_x0020_Date`. That makes it more usable across widgets where `data` may be structured differently.

Reply:
```
formattedDate = dateFormat(data.newsarticle.Publish_x0020_Date)
data.newsarticle.publisheddate = formattedDate
```